app/services/vot/authn_context_class_ref_component_values.rb

```diff
@@ -1,35 +1,53 @@
 # frozen_string_literal: true
 
 module Vot
-  module LegacyComponentValues
+  module AuthnContextClassRefComponentValues
     ## Identity proofing ACR values
+
+    # @deprecated - Use IAL1
     LOA1 = ComponentValue.new(
       name: Saml::Idp::Constants::LOA1_AUTHN_CONTEXT_CLASSREF,
-      description: 'Legacy LOA1',
+      description: 'Legacy LOA1 - no identity proofing',
       implied_component_values: [],
       requirements: [],
     ).freeze
+    # @deprecated - Use IAL2
     LOA3 = ComponentValue.new(
       name: Saml::Idp::Constants::LOA3_AUTHN_CONTEXT_CLASSREF,
-      description: 'Legacy LOA3',
+      description: 'Legacy LOA3 - identity proofing is performed',
       implied_component_values: [],
       requirements: [:aal2, :identity_proofing],
     ).freeze
+
     IAL1 = ComponentValue.new(
       name: Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF,
-      description: 'Legacy IAL1',
+      description: 'IAL1 - no identity proofing (rev 3)',
       implied_component_values: [],
       requirements: [],
     ).freeze
     IAL2 = ComponentValue.new(
       name: Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF,
-      description: 'Legacy IAL2',
+      description: 'IAL2 - identity proofing is performed (rev3)',
       implied_component_values: [],
       requirements: [:aal2, :identity_proofing],
     ).freeze
+    IAL2_BIO_REQUIRED = ComponentValue.new(
+      name: Saml::Idp::Constants::IAL2_BIO_REQUIRED_AUTHN_CONTEXT_CLASSREF,
+      description: 'IAL2 - identity proofing with biometric comparison (rev3)',
+      implied_component_values: [],
+      requirements: [:aal2, :identity_proofing, :biometric_comparison,
+                     :two_pieces_of_fair_evidence, :biometric_comparison_required],
```

Comment:
@Sgtpluck - I see `:biometric_comparison_required` here, but it isn't used in `AuthnContextResolver`, nor is it an boolean attribute on `Vot::Parser::Result`. If I understand the original design correctly, the `ComponentValue.requirements` and attributes ending with `?` on the `Result` object should be the same.

(The only other place I could find `:biometric_comparison_required` used in production code is in `ServiceProviderRequest`, which seems totally unrelated.)

Reply:
oh yeah, i was trying something with this that i didn't end up liking! that can be removed.